### ISSUE TITLE
Estimated SSB

### ISSUE THREAD

**Issue by mle2718** [lkerr/groundfish-MSE#288]
Hi Jerelle and Mackenzie,
I'm trying to figure out which object represents the estimated SSB that is used by the manager to set the catch limit according to the harvest control rule.

I'm pretty sure SSB is the "true" SSB.

I thought I was looking for SSBest 
https://github.com/lkerr/groundfish-MSE/blob/bfba39b6fd244d2dd5e1bb92dac9ba9225f99b74/functions/datawrangling/get_fillRepArrays.R#L87

However, it has the wrong dimensions of output.  It is a 2D array of (replicates and years), instead of a 3D array (replicates, management, and years).

I also saw SSBPROXY and SSBPROXYT, but I'm not really sure what either of these are.  Can you help me figure out which object to use?

**Comment by jerellejesse** [comment]
The SSBPROXY is the SSB at Fmsy proxy (F40%). SSBPROXYT is the true (OM) metric. I don't think that is what you want. I will look into the SSBest container because it looks like it is saving the SSBest each year of the assessment for each year of the sim. It doesn't seem like it is set up to save for reps or mprocs? I'll dig in a little more.

**Comment by mle2718** [comment]
@jerellejesse Thanks! Def not what I want.  Do you think this:
https://github.com/lkerr/groundfish-MSE/blob/bfba39b6fd244d2dd5e1bb92dac9ba9225f99b74/functions/datawrangling/get_containers.R#L174-L177

is part of the problem. I was expecting it to look like:
```
 SSBest = save_vector_ann, 
```

**Comment by jerellejesse** [comment]
Yes, I think that is the problem but I did some testing and it wasn't quickly fixed. I think it is because the assessment saves more info at each time step. At each sim year we save the SSB estimate for the number of years that are included in the assessment not just the terminal estimate. Maybe an easy fix but I haven't tried yet

**Comment by mle2718** [comment]
That makes sense, I think I saw some code about Mohn's rho right next to it. 

Honestly, this is lower priority.  

I can look at the survey indices and see if they are all the same or different.  If the survey indices different across replictes in the same year, then I can infer that the estimated SSB is different, and that's what's causing the different F_fullAdvices.
